glib-macros/Cargo.toml

```diff
@@ -16,7 +16,7 @@ heck = "0.3"
 proc-macro-error = "1.0"
 proc-macro2 = "1.0"
 quote = "1.0"
-syn = "1.0"
+syn = { version = "1.0", features = ["full"] }
```

Comment:
It's probably better to only opt-in to the features we actually use instead of enabling all :) See also https://github.com/gtk-rs/gtk-rs/issues/41

Reply:
According to https://docs.rs/syn/1.0.61/syn/enum.ImplItem.html:
> This type is available only if Syn is built with the "full" feature.

I'm not that familiar with Syn, but looking at it's `Cargo.toml`, "full" doesn't actually imply any other features, somewhat counterintuitively. It [actually](https://docs.rs/syn/1.0.61/syn/index.html#optional-features) means this:
> full — Data structures for representing the syntax tree of all valid Rust source code, including items and expressions.

Reply:
How confusing